dnf-behave-tests/dnf/security-advisory.feature

```diff
@@ -43,32 +47,56 @@ Scenario: upgrade cves
         | upgrade       | advisory_B-0:1.0-4.x86_64 |
 
 
-# @dnf5
-# TODO(nsella) Unknown argument "--sec-severity" for command "upgrade-minimal"
 Scenario: upgrade-minimal sec-severity
-   When I execute dnf with args "upgrade-minimal --sec-severity Moderate"
+   When I execute dnf4 with args "upgrade-minimal --sec-severity Moderate"
+   When I execute dnf5 with args "upgrade --minimal --advisory-severities=Moderate"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                   |
+        | upgrade       | advisory_B-0:1.0-2.x86_64 |
+
+
+Scenario: upgrade-minimal with pkgs specified sec-severity
+   When I execute dnf4 with args "upgrade-minimal advisory_B --sec-severity Moderate"
+   When I execute dnf5 with args "upgrade --minimal advisory_B --advisory-severities=Moderate"
    Then the exit code is 0
     And Transaction is following
         | Action        | Package                   |
         | upgrade       | advisory_B-0:1.0-2.x86_64 |
 
 
-# @dnf5
-# TODO(nsella) Unknown argument "--sec-severity" for command "upgrade"
 Scenario: upgrade secseverity
-   When I execute dnf with args "upgrade --sec-severity Critical"
+   When I execute dnf4 with args "upgrade --sec-severity Critical"
+   When I execute dnf5 with args "upgrade --advisory-severities Critical"
```

Comment:
Have we agreed on a singular form of the options? `--advisory-severity`?

Reply:
We didn't, I chose the approach where options that accept a list have a plural form (`--advisory-severities=critical,moderate`, `--cves=CVE1,CVE2`, `--bzs=BZ1,BZ2..`,..) with some compatibility aliases such as: `--cve`, `--bz` - they are still available but not auto-completed.

Reply:
Ok, thanks for an explanation.